independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/InjectionPointInfo.java

```diff
@@ -282,6 +304,16 @@ public String getTargetInfo() {
         if (target == null) {
             return "";
         }
+        if (methodParameterTarget != null && Kind.METHOD_PARAMETER.equals(methodParameterTarget.kind())) {
+            String method = methodParameterTarget.asMethodParameter().method().name();
+            if (method.equals(Methods.INIT)) {
+                method = " constructor";
+            } else {
+                method = "#" + method + "()";
+            }
+            return "parameter '" + methodParameterTarget.asMethodParameter().name() + "' of "
+                    + methodParameterTarget.asMethodParameter().method().declaringClass().name() + method;
+        }
```

Comment:
This entire added piece seems useless?

Reply:
WDYM? The class now *potentially* holds two annotation targets. So we first try to create the String representation with the method param AT and if not present, we fallback to the original target.

Reply:
But the original code (below this new one) does the same thing, IIUC?

Reply:
Ah, I see what you mean. Yes, it will generate the same String outcome.
I wanted to cover the `METHOD_PARAMETER` case so that it won't be forgotten once we remove `getTarget`.

But you're right, I can remove it and just keep a comment.

Reply:
I have added this code under `case METHOD_PARAMETER` which won't do anything for now but will work as intended once we act on the deprecation removal.